### PR TITLE
Update risk level feedback

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard14**
Remove page from Sanity and from the query
Switched statement for showing the correct text

### CODE REVIEW

File: packages/app/src/domain/escalation-level/escalation-level-label.tsx [hunk 1]
```diff
@@ -54,9 +54,9 @@ export function EscalationLevelLabel({
         <InlineText variant="body2" color="body">
           {replaceVariablesInText(
             validFrom === lastCalculated
-              ? siteText.national_escalation_levels.valid_from
-              : siteText.national_escalation_levels
-                  .valid_from_and_last_calculated,
+              ? siteText.national_escalation_levels
+                  .valid_from_and_last_calculated
+              : siteText.national_escalation_levels.valid_from,
```
[the remark this answers is not in the document]
Comment (VWSCoronaDashboard14) [reply packages/app/src/domain/escalation-level/escalation-level-label.tsx:59]: Oh shoot

[the remark this answers is not in the document]
Comment (VWSCoronaDashboard14) [reply packages/app/src/domain/escalation-level/escalation-level-label.tsx]: I switched on production and dev Lokaise the texts, now it should work 😓

[the remark this answers is not in the document]
Comment (VWSCoronaDashboard16) [reply packages/app/src/domain/escalation-level/escalation-level-label.tsx]: Wait no the text was also entered correctly

[the remark this answers is not in the document]
Comment (VWSCoronaDashboard16) [reply packages/app/src/domain/escalation-level/escalation-level-label.tsx]: So it should have worked already

[the remark this answers is not in the document]
Comment (VWSCoronaDashboard16) [reply packages/app/src/domain/escalation-level/escalation-level-label.tsx]: Can we compare the dates directly? Maybe they are the same day but not the exact same date

**Pull request merged**